cmds/core/ping/ping.go

```diff
@@ -166,8 +166,10 @@ func ping(host string) error {
 	p := New()
 	// ping needs to run forever, except if '*iter' is not zero
 	waitFor := time.Duration(*wtf) * time.Millisecond
-	for i := uint64(0); i <= *iter; i++ {
```

Comment:
if you make the default value of iter be a very large number, instead of 0, nothing else will change. Would that be OK?

Reply:
Sure, it will be better.